src/Cli/Microsoft.TemplateEngine.Cli/Commands/details/DetailsCommand.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Cli.NuGet;
+using Microsoft.TemplateEngine.Edge.Settings;
+
+namespace Microsoft.TemplateEngine.Cli.Commands
+{
+    internal class DetailsCommand : BaseCommand<DetailsCommandArgs>
+    {
+        private static NugetApiManager _nugetApiManager = new NugetApiManager();
+
+        internal DetailsCommand(
+            Func<ParseResult, ITemplateEngineHost> hostBuilder)
+            : base(hostBuilder, "details", SymbolStrings.Command_Details_Description)
+        {
+            AddArgument(NameArgument);
+            AddOption(VersionOption);
+            AddOption(InteractiveOption);
+            AddOption(AddSourceOption);
+        }
+
+        internal static Argument<string> NameArgument { get; } = new("package-identifier")
+        {
+            Description = LocalizableStrings.DetailsCommand_Argument_PackageIdentifier,
+            Arity = new ArgumentArity(1, 1)
+        };
+
+        internal static Option<string> VersionOption { get; } = new Option<string>(new string[] { "-version", "--version" })
+        {
+            Description = LocalizableStrings.DetailsCommand_Option_Version,
+            Arity = new ArgumentArity(0, 1)
```

Comment:
@vlada-shubina , does this property represent the min/max number of params?

Reply:
it represents min-max amount of accepted args for the options. For this option it has to be 1,1